deploy/crds/applicationmonitoring.integreatly.org_applicationmonitorings_crd.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: applicationmonitorings.applicationmonitoring.integreatly.org
+spec:
+  group: applicationmonitoring.integreatly.org
+  names:
+    kind: ApplicationMonitoring
+    listKind: ApplicationMonitoringList
+    plural: applicationmonitorings
+    singular: applicationmonitoring
+  scope: ""
+  validation:
+    openAPIV3Schema:
+      description: ApplicationMonitoring is the Schema for the applicationmonitorings
+        API
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: ApplicationMonitoringSpec defines the desired state of ApplicationMonitoring
+          properties:
+            additionalScrapeConfigSecretKey:
+              type: string
+            additionalScrapeConfigSecretName:
+              type: string
+            alertmanagerInstanceNamespaces:
+              type: string
+            labelSelector:
+              description: 'INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+                Important: Run "operator-sdk generate k8s" to regenerate code after
+                modifying this file'
+              type: string
+            prometheusInstanceNamespaces:
+              type: string
+            prometheusRetention:
+              type: string
+            prometheusStorageRequest:
+              type: string
+          required:
+          - alertmanagerInstanceNamespaces
+          - labelSelector
+          - prometheusInstanceNamespaces
```

Comment:
Are these all required?

Reply:
Yep, these values are required to ensure that the namespace that Prometheus operator is watching is limited, otherwise it might interfere with the openshift-monitoring stack.